patches/react-native-svg-transformer+1.0.0.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/react-native-svg-transformer/index.js b/node_modules/react-native-svg-transformer/index.js
+index aac1ce1..22cf73a 100644
+--- a/node_modules/react-native-svg-transformer/index.js
++++ b/node_modules/react-native-svg-transformer/index.js
+@@ -1,6 +1,6 @@
+ const { resolveConfig, transform } = require("@svgr/core");
+ const resolveConfigDir = require("path-dirname");
+-const upstreamTransformer = require("metro-react-native-babel-transformer");
++const upstreamTransformer = require("@react-native/metro-babel-transformer");
```

Comment:
Curious if this is fixed in latest version 1.1.0

Reply:
No, it isn't :/ https://github.com/kristerkari/react-native-svg-transformer/blob/e1bffad71f94bb6f0b98fd9e24a2d93cff717fa6/index.js#L3. But there is a PR for that: https://github.com/kristerkari/react-native-svg-transformer/pull/312/files